src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "AppKeyBindingsSerialization.h"
+#include "KeyChordSerialization.h"
+#include "Utils.h"
+#include <winrt/Microsoft.Terminal.Settings.h>
+
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::TerminalApp;
+
+static constexpr std::string_view KEYS_KEY{ "keys" };
+static constexpr std::string_view COMMAND_KEY{ "command" };
+
+static constexpr std::wstring_view COPYTEXT_KEY{ L"copy" };

```

Comment:
Is there a lot of back/forth because these are `w` and not narrow?

Reply:
If these are never used outside of a utf-8 context, they shouldn't need to be wide